### PR TITLE
FIX map or array type conversion bug.

### PR DESCRIPTION
**Pull request by y-matsuwitter**
I fixed issue #3.

Remove `(.+?, .+?)` from  a `explain` query result like `data:map(varchar, varchar)` .

### CODE REVIEW

**Comment by toyama0919** [comment]
Oh, thanks.

File: lib/embulk/input/presto/explain_parser.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ def self.parse(explain_result)
           explain_text = explain_result.flatten.last.lines.first
           column_name_raw, column_type_raw = explain_text.split(' => ')
           names = column_name_raw.split('[').last.split(']').first.split(',').map{ |name| name.strip }
-          types = column_type_raw.split('[').last.split(']').first.split(',').map{ |info| info.split(':').last }
+          types = column_type_raw.split('[').last.split(']').first.gsub(/\(.+?, .+?\)/, "").split(',').map{ |info| info.split(':').last }
```
Comment (toyama0919) [inline lib/embulk/input/presto/explain_parser.rb]: I think better this.

```
.gsub(/\(.*/, '')
```

Comment (toyama0919) [reply lib/embulk/input/presto/explain_parser.rb]: I think that you can delete all  '(' behind

Comment (y-matsuwitter) [reply lib/embulk/input/presto/explain_parser.rb]: `/\(.*/` will remove some of the needed schema unintentionally.

For example:

```
"data:map(varchar, varchar), user_id:bigint".gsub(/\(.*/, '')
=> "data:map"
```

Above example should be `"data:map, user_id:bigint"`.

But if the schema contains an `array` type, my PR  is not correct. 
So I'll use `/\(.*?\)` instead.

```
irb(main):003:0> "data:map(varchar, varchar), user_id:bigint, data2:array(double)".gsub(/\(.*?,.*?\)/, '')
=> "data:map, user_id:bigint, data2:array(double)"
irb(main):004:0> "data:map(varchar, varchar), user_id:bigint, data2:array(double)".gsub(/\(.*?\)/, '')
=> "data:map, user_id:bigint, data2:array"
```

**Comment by toyama0919** [comment]
LGTM.

**Comment by toyama0919** [comment]
released.

embulk-input-presto | RubyGems.org | your community gem host https://rubygems.org/gems/embulk-input-presto/versions/0.2.5

**Pull request merged**